libs/api/repository/src/lib/gn4/platform/gn4-platform.mapper.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  MeResponseApiModel,
+  UserApiModel,
+} from '@geonetwork-ui/data-access/gn4'
+import { UserModel } from '@geonetwork-ui/common/domain/model/user/user.model'
+import { Injectable } from '@angular/core'
+import { AvatarServiceInterface } from '../auth/avatar.service.interface'
+
+@Injectable()
+export class Gn4PlatformMapper {
```

Comment:
I feel like these mappers could be simple pure functions exported as is, instead of an Angular service.

Reply:
Yes I actually wanted to create a static class, but the mappers need for instance the `AvatarService` which is injected.